terraform/healthcheck/lambda.tf

```diff
@@ -0,0 +1,89 @@
+# Add required providers
+terraform {
+  required_providers {
+    archive = {
+      source = "hashicorp/archive"
+      version = "2.4.2"
+    }
+  }
+}
+
+# Creates iam role
+resource "aws_iam_role" "healthcheck" {
+  name               = "${var.environment}-healthcheck"
+  assume_role_policy = data.aws_iam_policy_document.assume_role.json
+}
+
+# This attaches the policy needed for logging to the lambda's IAM role. #3
```

Comment:
#3 ?

Reply:
Thas a marker for me that i forgot to remove sorry it is gone now.